db/seeds/scope_limitations.rb

```diff
@@ -0,0 +1,18 @@
+require 'csv'
+
+file_path = Rails.root.join('db', 'seeds', 'scope_limitations.csv')
+
+data = CSV.read(file_path)
+data.shift
```

Comment:
If you use [the `headers: true` option of `CSV.read`](http://ruby-doc.org/stdlib-2.6.3/libdoc/csv/rdoc/CSV.html#class-CSV-label-CSV+with+headers), the code here can be simplified a lot. I think this will work:
```ruby
CSV.read(file_path, header:true, header_converters: :symbol).do |row|
  scope_limitation = ScopeLimitation.where(code: row[:code]).first_or_initialize
  scope_limitation.update! row.to_h
end
```